### ISSUE TITLE
Plugin 0.8.6 (with Relodium 0.9.3) breaks with PyCharm 2022.2.3

### ISSUE THREAD

**Issue by laurapons** [reloadware/reloadium#52]
**Describe the bug**
Relodium breaks.
I had relodium installed and upgraded both PyCharm and Relodium versions.
After the upgrade, the plugin fails when running. Because code is obfuscated I cannot see where it breaks, but I attached the log console.

**Screenshots**
![image](https://user-images.githubusercontent.com/1275184/196942123-1f93040e-b588-4df1-92c5-c8c306d2be77.png)

**Desktop (please complete the following information):**
 - OS: Windows 11
 - Reloadium package version: 0.8.6
 - Editor: PyCharm 2022.2.3 (Professional Edition) Build #PY-222.4345.23, built on October 10, 2022
 - Run mode: Run

**Additional context**

**Comment by dkrystki** [comment]
@laurapons Thanks for posting this. Could you please paste the traceback as plain text?

**Comment by laurapons** [comment]
`````
Traceback (most recent call last):
  File "C:\Users\user\AppData\Local\Programs\Python\Python38\lib\runpy.py", line 194, in _run_module_as_main
    return _run_code(code, main_globals, None,
  File "C:\Users\user\AppData\Local\Programs\Python\Python38\lib\runpy.py", line 87, in _run_code
    exec(code, run_globals)
  File "C:\Users\user\.reloadium\package\3.8\reloadium\__main__.py", line 5, in <module>
    start(sys.argv)
  File "C:\Users\user\.reloadium\package\3.8\reloadium\corium\l1ll111l1111l111Il1l1.py", line 615, in start
  File "C:\Users\user\.reloadium\package\3.8\reloadium\corium\l1ll111l1111l111Il1l1.py", line 599, in llll11lll1l1ll1lIl1l1
  File "C:\Users\user\.reloadium\package\3.8\reloadium\corium\l1ll111l1111l111Il1l1.py", line 565, in l1l1l11l111l11llIl1l1
  File "C:\Users\user\.reloadium\package\3.8\reloadium\corium\l1ll111l1111l111Il1l1.py", line 249, in ll11l111lll1111lIl1l1
  File "C:\Users\user\.reloadium\package\3.8\reloadium\corium\lll11ll1llll11llIl1l1.py", line 544, in ll11l111lll1111lIl1l1
  File "C:\Users\user\.reloadium\package\3.8\reloadium\corium\lll11ll1llll11llIl1l1.py", line 550, in l11111lll11l11l1Il1l1
  File "C:\Users\user\.reloadium\package\3.8\reloadium\corium\l1ll111l1111l111Il1l1.py", line 212, in ll1l1lll11l11ll1Il1l1
  File "D:\a\reloadware\reloadware\reload\package\__obfuscated__\reloadium\fast\lll1l1l111l1ll11Il1l1\l1lll1ll111l1111Il1l1.py", line 722, in ll1l1lll11l11ll1Il1l1
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "C:\Users\user\AppData\Local\Programs\Python\Python38\lib\multiprocessing\managers.py", line 29, in <module>
    from . import connection
  File "C:\Users\user\.reloadium\package\3.8\reloadium\corium\l1l1l1l1lll111llIl1l1.py", line 141, in l11ll1lllllll1l1Il1l1
ImportError: attempted relative import with no known parent package

Loaded 184 watched modules so far from paths:
 - D:\Dev\**\*.py


```

**Comment by dkrystki** [comment]
Thanks for the traceback however I am unable to reproduce it.
Do you think you could provide a full reproducer (a smallest code example that would reproduce it)?
I'm pretty sure it's something around importing and using multiprocessing library.

0.9.3 added support for multiprocessing https://github.com/reloadware/reloadium/issues/40 and the example provided there works perfectly now.

**Comment by laurapons** [comment]
I will try to provide a small example (app is too long). In the meantime, is there any way to use the previous version 0.9.2 from the plugin?

**Comment by dkrystki** [comment]
Follow these steps to downgrade:
1) Uninstall the plugin
2) Remove <home>/.reloadium/package directory
3) Install 0.9.2 plugin

**Comment by laurapons** [comment]
On the marketplace there is up to 0.8.6, which installs Reloadium 0.9.3

**Comment by dkrystki** [comment]
Try downloading it from there https://plugins.jetbrains.com/plugin/18509-reloadium--hot-reloading--profiling-for-python/versions/stable/218512 and dragging the file onto PyCharm

**Comment by dkrystki** [comment]
Fixed in reloadium 0.9.4 and PyCharm plugin 0.8.8

**Comment by laurapons** [comment]
Up & running, thanks for the quick fixing :)
